jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/tim/TimProcessedJsonConverter.java

```diff
@@ -0,0 +1,1777 @@
+        double centerLat = FieldConversions.convertLat(center.getLat().getValue());
+        double centerLon = FieldConversions.convertLong(center.getLong_().getValue());
+        double radius = circle.getRadius().getValue();
+
+        return createCircleApproximation(centerLon, centerLat, radius);
+    }
+
+    /**
+     * Create a circle approximation using multiple points.
+     *
+     * @param centerLon Center longitude
+     * @param centerLat Center latitude
+     * @param radius Radius in meters
+     * @return List of coordinate points forming the circle
+     */
+    private List<List<Double>> createCircleApproximation(double centerLon, double centerLat, double radius) {
+        List<List<Double>> coordinates = new ArrayList<>();
+
+        for (int i = 0; i <= CIRCLE_APPROXIMATION_POINTS; i++) {
+            double angle = 360.0 * i / CIRCLE_APPROXIMATION_POINTS; // Angle in degrees
+            List<Double> point = calculateDestinationPoint(centerLon, centerLat, angle, radius);
+            coordinates.add(point);
+        }
+
+        return coordinates;
+    }
+
+    /**
+     * Calculate destination point given bearing and distance using proper geodetic calculations
+     */
+    private List<Double> calculateDestinationPoint(double lon, double lat, double bearing, double distanceMeters) {
+        Coordinate destCoord = GeodeticUtils.calculateDestination(lon, lat, bearing, distanceMeters);
+        return Arrays.asList(destCoord.x, destCoord.y);
+    }
+
+    /**
+     * Create a rectangle from an anchor point with default padding.
+     *
+     * @param anchor The anchor position
+     * @return List of coordinate points forming a rectangle
+     */
+    private List<List<Double>> createRectangleFromAnchor(Position3D anchor) {
+        double lat = FieldConversions.convertLat(anchor.getLat().getValue());
+        double lon = FieldConversions.convertLong(anchor.getLong_().getValue());
+
+        return createRectangleCoordinates(lon, lat, DEFAULT_PADDING_DEGREES);
+    }
+
+    /**
+     * Create rectangle coordinates around a center point.
+     *
+     * @param centerLon Center longitude
+     * @param centerLat Center latitude
+     * @param padding Padding in degrees
+     * @return List of coordinate points forming a rectangle
+     */
+    private List<List<Double>> createRectangleCoordinates(double centerLon, double centerLat, double padding) {
+        List<List<Double>> coordinates = new ArrayList<>();
+
+        coordinates.add(Arrays.asList(centerLon - padding, centerLat - padding));
+        coordinates.add(Arrays.asList(centerLon + padding, centerLat - padding));
+        coordinates.add(Arrays.asList(centerLon + padding, centerLat + padding));
+        coordinates.add(Arrays.asList(centerLon - padding, centerLat + padding));
+        coordinates.add(Arrays.asList(centerLon - padding, centerLat - padding)); // Close the rectangle
+
+        return coordinates;
+    }
```

Comment:
suggestion: JTS [GeometricShapeFactory](https://locationtech.github.io/jts/javadoc/org/locationtech/jts/util/GeometricShapeFactory.html) can also create rectangles and would work easily to simplify this code, with no coordinate conversion needed since the padding and lat/lon are all in the same units (degrees)